pages/common/k6.md

```diff
@@ -0,0 +1,27 @@
+# k6
+
+> A modern load testing tool for developers and testers in the DevOps era.
+
+- Run load test locally:
+
+`k6 run {{script.js}}`
+
+- Run load test locally with set number of virtual users and set duration:
```

Comment:
"with a given number of virtual users and duration"

Reply:
changed in 4cbf5349